importers/deb/deb_test.go

```diff
@@ -0,0 +1,434 @@
+				"file.04": "0a632850049f80763ada81ec0cacf015dbd67fb1b956ec2acb8aa862e511b3bc",
+				"file.05": "471ed5b2f059540ae4d8586a47329272c774561fc1a9c15fb22f2d7939b6820b",
+				"file.06": "b8862d9e62c15c73527ca72b4e5e85809d4254326800eb2c65b35339029e02d1",
+				"file.07": "733dac1b5272600122760db1f917bb4729d69e5a69f039797f75e685a1152bf0",
+				"file.08": "c8c6fb7771c086f786ea8d60bf703bbeea3ed7984ce962493994493839f70713",
+				"file.09": "d5de3fe6a4559c59ad103ab40e01c4fc0df7eb8ba901d50e5ceae3909b2e0d61",
+				"file.10": "05d1223b8b287684ce015d918b90555f9cb4c753558d511d179ada227820f6ba",
+			},
+		},
+		{
+			image: "ubuntu-laptop.deb",
+			files: map[string]string{
+				"file.01": "db391f171b17f750e440c6311b8ef618705d14f0de99eedff954b318967d9e1a",
+				"file.02": "f99254f3f48cf49195f0de6ce59257634e2c9a1c251c7013d7ba246ed6ec841d",
+				"file.03": "d15a7d5f9731738452cd67b6bfa035e4b12b3d934d3b47d37c1c8273eeaa23fd",
+				"file.04": "15a11e8e69ba8203df18c1f2baaee4ffb7de4be7d4abb4bd86d7e232231aeaae",
+				"file.05": "c84f7f5db9e404bafe87bc4123474801486ad8f65ca63de4f790bb885c99e25d",
+				"file.06": "ff0ca17a9756bb3d91b748887ef2d5fe7e97137e3f5b1f6b8ec6cc7fe369f8fd",
+				"file.07": "229faa1edf71fa3b221d17f5d90d852029c2559f91f53eb464036d9b0a197eee",
+				"file.08": "c8fc201d41aeb679d8da4708491e7ed4ba3a4e3be85c50210b08a2372284624a",
+				"file.09": "ba1b175b82ea3493bb2c0c0bca529aaeb42ce5cf6888cb09f7df8e87a639beae",
+				"file.10": "7ca80a863d5c935068511d939ad8857f6c0e1fe1bf62ff4160b1a78bba84e3ba",
+			},
+		},
+		{
+			image: "ubuntu-server.deb",
+			files: map[string]string{
+				"file.01": "9076527a1200b53570b8719f0049b8544c7366003c355cd740858fd3569729f6",
+				"file.02": "78d78a3d686c9580a0fb123894f16156a822261a30499f3e8d34cad77496e0fc",
+				"file.03": "ebb15f223ca81484e714a08ff93689189214fbb1003f12e007fd8deb98fd6b35",
+				"file.04": "cd1be623861c325203d1582826a5bffdbc732fa5f86984d8422c2623d45c3164",
+				"file.05": "207b2c6bc867f366e2f7d7607662bd9329e84c68a97affba23963e9a4245ef01",
+				"file.06": "f481bbd1719e1adb5446b1cb0df10b7564edfdc170cc5e2afa4b72b74a56788c",
+				"file.07": "807034689f2c2b179b60692162bf99eb390d39a2b39e6897949b030805e953d2",
+				"file.08": "3665d702279c54659944ba9def060306f261d2706793ddbc9f6214a377c36a4c",
+				"file.09": "ebd093091d65f012ebb1c0d6128334984924132e865a98b6fa6349f143102811",
+				"file.10": "40998bbd7e456cb57beacc7006eb5c949160527fb6589c8fce3b1f49a1a9f3bd",
+			},
+		},
+	}
+
+	if !cmp.Equal(wantExtractions, gotExtractions, cmp.AllowUnexported(extraction{})) {
+		t.Errorf("Preprocess() unexpected diff (-want/+got):\n%s", cmp.Diff(wantExtractions, gotExtractions, cmp.AllowUnexported(extraction{})))
+	}
+}
+
+func TestImageFunctions(t *testing.T) {
+	id := "ubuntu-desktop.deb"
+	repoPath := "/tmp/glinux-repo"
+	localTarGzPath := "/tmp/glinux-repo/20200108.00.00-ubuntu-desktop.deb"
+	remotePath := "/x20/glinux-repo/20200108.00.00-ubuntu-desktop.deb"
```

Comment:
Please rename the glinux/x20 strings

Reply:
Done. Note that these tests were copy-pasted from the targz importer which still contains x20/glinux references.